src/styles/index.scss

```diff
@@ -0,0 +1,13 @@
+@import './blocks/variables';
+@import './blocks/reset';
+@import './blocks/header';
+@import './blocks/products';
+@import './blocks/stars';
+@import './blocks/card';
+
+.page {
+  &__body {
+    margin: 0;
+    font-family: Roboto, Helvetica, sans-serif;
+  }
+}
```

Comment:
good practice is to leave 1 line at the end of the file (please do this fix in each your file)